buildSrc/build.gradle.kts

```diff
@@ -28,6 +28,8 @@ dependencies {
   implementation(gradleApi())
   implementation(localGroovy())
 
+  implementation("io.opentelemetry.instrumentation.muzzle-generation:io.opentelemetry.instrumentation.muzzle-generation.gradle.plugin:0.7.0")
```

Comment:
If I am not mistaked these plugins should substitute `MuzzlePlugin.groovy` so could we remove `MuzzlePlugin.groovy`?

Reply:
I, unfortunately, wasn't able to drop in the plugin class as a replacement yet, but looking forward to doing that as we incrementally migrate these modules to more standard patterns!